docs/superpowers/specs/2026-07-27-visual-refresh-design.md

```
@@ -0,0 +1,284 @@
+  graphic (a "3" over a conic-gradient wedge, which read as decorative-cryptic
+  rather than informative). `.moat-pie` was its only consumer and `--chart-2`
+  was `.moat-pie`'s only consumer, so both were deleted along with
+  `--color-chart-2`. No retune needed.
+
+## Design
+
+### 1. Color tokens (`app/globals.css`)
+
+Light mode:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)